src/test/resources/com/cloudbees/jenkins/plugins/advisor/casc/configuration-as-code.yaml

```diff
@@ -0,0 +1,8 @@
+advisor:
+  acceptToS: true
+  cc: "we@email.com,they@email.com"
```

Comment:
would this be better as a list?

Reply:
Yes, yes it would 😓

Reply:
FWIT, the README file says how to set it programmatically: https://github.com/jenkinsci/cloudbees-jenkins-advisor-plugin#configure-programmatically

Reply:
Since you’re writing your own configuration you don’t have to follow the UI and can choose the yaml you want, but up to you, just a thougt

Reply:
Since you’re writing your own configuration you don’t have to follow the UI and can choose the yaml you want, but up to you, just a thougt

Reply:
@timja you're right. What I mean is that I opted to mimic the exact configuration that an admin user will introduce from the UI when the same admin user has to write/edit the yaml file. Just a try to ease life for users.
@aheritier as @timja says, the configurator is a new class, so we are in time to modify the yaml structure.

Reply:
I don't know :-)
Both arguments are right. Like @timja and @casz I prefer in the YAML to have a list.
For sure your copy/paste argument @fcojfernandez is relevant too
I think that in the future I will see if I cannot improve the UI to allow to add cc emails one by one with a (+) to have something more coherent.
Let's use in list if it's not difficult and I will change the UI later. I think having a good datamodel which is stable over the time is better

Reply:
I've been looking into it. Changing the _describe_ and _instance_ methods it not enough. At some point the [BaseConfigurator.configure method](https://github.com/jenkinsci/configuration-as-code-plugin/blob/2f251fcd34f5a26fcc6c0784195cb489d035b72e/plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java#L299-L367) is executed and since it's calling the [_Set describe_ method](https://github.com/jenkinsci/configuration-as-code-plugin/blob/2f251fcd34f5a26fcc6c0784195cb489d035b72e/plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java#L56-L75), it will get the `cc` field as String and it will try to configure it that way, causing an error. It's needed to override the _Set describe_ or the _configure_ methods. Taking into account that you plan to change the UI, I'm not sure it's worth to do that override.